frontend/src/prompted_readings/analysis_view.js

```diff
@@ -1,4 +1,63 @@
 import React from "react";
+import PropTypes from 'prop-types';
+
+class FrequencyFeelingTable extends React.Component {
+    render() {
+        return (
+            <div>
+                <h1>Frequency Feelings</h1>
+                <table border="1" cellPadding="5">
+                    <tbody>
+                        <tr>
+                            <th>Word</th>
+                            <th>Frequency</th>
+                        </tr>
+                        {this.props.feelings.map((el, i) => (
+                            <tr key={i}>
+                                <td>{el[0]}</td><td>{el[1]}</td>
+                            </tr>
+                        ))}
+                    </tbody>
+                </table>
+            </div>
+        )
+    }
+}
+
+FrequencyFeelingTable.propTypes = {
+    feelings: PropTypes.array,
+};
+
+class ContextVsViewTime extends React.Component {
+    render() {
+        const viewTimesList = Object.entries(this.props.viewTime);
+        const roundedViewTimes = viewTimesList.map(context =>
+            [context[0] , Math.round(context[1] * 1000) / 1000]);
+        return (
+            <div>
+                <h1>Mean View Times of Different Contexts</h1>
+                <table border="1" cellPadding="5">
```

Comment:
Same deal as my other comment -- let's replace this with Bootstrap. Not blocking this merge, but a good next step. 
https://getbootstrap.com/docs/4.3/content/tables/